src/Login/RegisterForm.tsx

```diff
@@ -50,31 +50,24 @@ export const RegisterForm: React.FC<RegisterFormProps> = () => {
 
     const handleRegister = async () => {
         try {
-            const resp = await AxiosRequest.post('/users/register', 
-                {
-                    email: formState.registerEmail,
-                    password: formState.registerPassword,
-                    firstName: formState.registerFirstName,
-                    lastName: formState.registerLastName,
-                });
-            logger.info(resp);
+            const resp = await registerUser({
+                email: formState.registerEmail,
+                password: formState.registerPassword,
+                firstName: formState.registerFirstName,
+                lastName: formState.registerLastName,
+            });
 
-            if (resp.status === 201) {
-                let message = 'Registration succeeded!';
-                if (!resp.data.data.verificationBypass) {
-                    message =`${message} Please check your email to continue.`;
-                }
-                setRegistrationAlert({message, variant: 'success'});
-            } else {
-                setRegistrationAlert({message: 'Registration failed.', variant: 'danger'});
+            if (resp.status !== 201) {
+                logger.warn(`handleRegister: Registration succeeded however got unexpected status code ${resp.status} instead of 201`);
+            }
```

Comment:
Do we even need this? We're not doing strict status code checking anywhere, so I think we can ditch it.